torch/nativert/executor/Placement.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <c10/core/Device.h>
+#include <c10/util/Logging.h>
+
+#include <optional>
+#include <unordered_map>
+
+namespace torch::nativert {
```

Comment:
Can you please add documentation for all of these as it is not very clear what the intended use of these is.

Reply:
> Can you please add documentation for all of these as it is not very clear what the intended use of these is.

@albanD added now!